processor/resourcedetectionprocessor/internal/resourcedetection.go

```diff
@@ -175,6 +179,21 @@ func getSerializableArray(inArr pdata.AnyValueArray) []interface{} {
 	return outArr
 }
 
+func MergeSchemaURL(currentSchemaURL string, newSchemaURL string) string {
+	if currentSchemaURL == "" {
+		return newSchemaURL
+	}
+	if newSchemaURL == "" {
+		return currentSchemaURL
+	}
+	if currentSchemaURL == newSchemaURL {
+		return currentSchemaURL
+	}
+	// TODO: handle the case when the schema URLs are different by performing
```

Comment:
Should we open an issue to track this?

Reply:
Filed an issue https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/4620